aikau/src/main/resources/alfresco/renderers/CommentsList.js

```diff
@@ -49,6 +49,33 @@ define(["dojo/_base/declare",
        * @default [{i18nFile: "./i18n/CommentsList.properties"}]
        */
       i18nRequirements: [{i18nFile: "./i18n/CommentsList.properties"}],
+
+      /**
+       * Whether to use a full-screen dialog when adding comments.
+       * 
+       * @instance
+       * @type {boolean}
+       * @default
+       * @since 1.0.43
+       */
+      addCommentsFullScreen: false,
```

Comment:
That's an interesting question. For the specific case we're looking at we almost want no padding at all so there's an argument that says we automatically set the padding to be 0... however, for future-proofing it might be worth exposing the additional argument.